### DISCUSSION TITLE
Design microphone arrays

### DISCUSSION THREAD

**Discussion by EzCww** [acoular/acoular#247] [Q&A] [answered]
Hi everyone. May I ask if Acoular has the ability to design arrays in addition to importing existing microphone coordinates?
Thanks.

**Comment by esarradj** [comment, marked as answer]
Presently, there is only the vague idea to implement some algorithm for Vogel's spiral and maybe even some other common array geometries.
If you just want to define the coordinates in the code instead of loading it from a file, this is already implemented and very simple. You can refer to this [example](https://www.acoular.org/examples/example_rotating_point_source.html) for a circular arrangement to see how this is done.
